build.gradle

```diff
@@ -165,3 +165,43 @@ startDatastore.doLast {
 tasks["check"].dependsOn startDatastore
 
 apply from: publishPlugin
+
+/**
+ * Adds a compile-time dependency on a Spine module.
+ *
+ * @param dependencyHandler the project dependency handler (basically {@code project.dependencies})
+ * @param artifactId the ID of the Spine module artifact
+ * @param version the Spine module version
+ * @param testScope tag parameter: {@code true} if the dependency should be added only for the tests
+ *                  (e.g. test utils modules`), {@code false} otherwise
+ */
+void dependOnSpine(final dependencyHandler,
```

Comment:
Let's not have that at all. It's way more lines than having just a list of dependencies explicitly.

Also, if you want to exclude some of transitive dependencies, you could have a collection in the same way and pass it to `excludes: ` section in each case. Or use global `configurations.all { ...}`.